blocks/init/src/Blocks/components/jumbotron/docs/story.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Fragment } from '@wordpress/element';
+import readme from './readme.mdx';
+import manifest from './../manifest.json';
+import imageManifest from './../../image/manifest.json';
+import headingManifest from './../../heading/manifest.json';
+import paragraphManifest from './../../paragraph/manifest.json';
+import buttonManifest from './../../button/manifest.json';
+import { JumbotronEditor } from '../components/jumbotron-editor';
+import { JumbotronOptions } from '../components/jumbotron-options';
+import { JumbotronToolbar } from '../components/jumbotron-toolbar';
+
+export default {
+	title: `Components/${manifest.title}`,
+	parameters: {
+		docs: { 
+			page: readme
+		}
+	},
+};
+
+const props = {
+	...imageManifest.example.attributes,
+	...headingManifest.example.attributes,
+	...paragraphManifest.example.attributes,
+	...buttonManifest.example.attributes,
+	...manifest.example.attributes,
+};
+
+export const editor = () => (
+	<JumbotronEditor {...props} />
+);
+
+export const options = () => (
+	<JumbotronOptions {...props} />
+);
+
+export const toolbar = () => (
+	<JumbotronToolbar {...props} />
+);
+
+const aligns = [
```

Comment:
Seems like this is repeating. Would it be a good idea to move it to a helper so that it can be reused?

Reply:
Totally agree. I hope once this component is live the core will provide the helper but I agree we should make one until then